pkg/reconciler/apiserversource/apiserversource.go

```diff
@@ -141,22 +137,27 @@ func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
 }
 
 func (r *Reconciler) reconcile(ctx context.Context, source *v1alpha1.ApiServerSource) error {
+	// This Source attempts to reconcile three things.
+	// 1. Determine the sink's URI.
+	//     - Nothing to delete.
+	// 2. Create a receive adapter in the form of a Deployment.
+	//     - Will be garbage collected by K8s when this CronJobSource is deleted.
+	// 3. Create the EventType that it can emit.
+	//     - Will be garbage collected by K8s when this CronJobSource is deleted.
 	source.Status.ObservedGeneration = source.Generation
 
 	source.Status.InitializeConditions()
 
-	track := r.resourceTracker.TrackInNamespace(source)
-
 	sinkObjRef := source.Spec.Sink
 	if sinkObjRef.Namespace == "" {
 		sinkObjRef.Namespace = source.Namespace
 	}
 
-	sourceDesc := source.Namespace + "/" + source.Name + ", " + source.GroupVersionKind().String()
+	sourceDesc := fmt.Sprintf("%s/%s,%s", source.Namespace, source.Name, source.GroupVersionKind().String())
```

Comment:
I don't know if it is important, but this skips a space after the comma in the original. 

I have no opinion on whether we want that change or not, just wanted to make sure you were aware of it.

Reply:
cronjobsource has it without space, that's why.